Library/Homebrew/test/testing_env.rb

```diff
@@ -6,7 +6,7 @@
 require "formulary"
 
 # Test environment setup
-%w[ENV Formula].each { |d| HOMEBREW_LIBRARY.join(d).mkpath }
+HOMEBREW_LIBRARY.join("Formula").mkpath
```

Comment:
Minor stylistic nit: Wouldn't `(HOMEBREW_LIBRARY/"Formula").mkpath` be more in line with what we tend to do elsewhere?